### PR TITLE
Comment out msxml3, msxml6, jscript9 in signing scenarios

### PR DESCRIPTION
**Pull request by brbrahm**
These three dlls are commented out in the rule definition section, so should also be commented out in the signing scenarios section in case people do not uncomment the first.

### CODE REVIEW

**Comment by brbrahm** [comment]
I am a WDAC feature team member. Please merge this change.

**Comment by e0i** [comment]
> I am a WDAC feature team member. Please merge this change.

cc @dansimp

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md [hunk 1]
```diff
@@ -888,9 +888,11 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
   <FileRuleRef RuleID="ID_DENY_WMIC"/>
   <FileRuleRef RuleID="ID_DENY_MWFC" /> 
   <FileRuleRef RuleID="ID_DENY_WFC" /> 
+  <!-- Uncomment the below if you have uncommented them in the above rule definition
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]:
```suggestion
  <!-- Uncomment the relevant line(s) below if you have uncommented them in the rule definitions above.
```

**Comment by illfated** [comment]
@brbrahm : If you are a Microsoft employee, it would be helpful to us other users if you add location information and a short biography description to your Github profile page.

**Comment by brbrahm** [comment]
@illfated thanks for the suggestion! I've updated my profile to reflect my MSFT employment

**Pull request merged**